command/repo.go

```diff
@@ -28,7 +28,7 @@ func init() {
 	repoCreateCmd.Flags().StringP("team", "t", "", "The name of the organization team to be granted access")
 	repoCreateCmd.Flags().Bool("enable-issues", true, "Enable issues in the new repository")
 	repoCreateCmd.Flags().Bool("enable-wiki", true, "Enable wiki in the new repository")
-	repoCreateCmd.Flags().Bool("public", false, "Make the new repository public")
+	repoCreateCmd.Flags().Bool("public", false, "Make the new repository public (default: private)")
```

Comment:
This makes sense, but since it's a boolean flag I think a more accurate help string is `(default: false)`. do you have an opinion, @ampinsk ?

Reply:
@vilmibm, if we will think about create and use the repository I guess that default: false is a good way because when we create the repository on Github.com  the public option is the default. I didn't remember that.

Reply:
I don't feel strongly either way but I have to think less when it says `default: private`. Totally hear that `default: false` fits better with our system though!

Reply:
@vilmibm sorry, I understand now what do you want to say with `(default: false)`, ignore my first reply. I totally agree with @ampinsk, I have to think less with `(default: private)`, however, as it's a boolean flag the value `(default: false)` is a good way. Can I close this pull request and open another with this change?

Reply:
`default: false` would be more "correct", but I agree with @ampinsk that `default: private` is better since you have you think less.